.circleci/config.yml

```diff
@@ -18,6 +18,7 @@ templates:
 
     - run: mkdir -p /tmp/artifacts/logs
     - run: npm run build
+    - run: npm run profiling-build
```

Comment:
Like this there is no profiling-build for linux-node-v10 which runs under `&wdio-steps`.

Do we need a profiling build for all node versions and for linux and osx? Or would it be enough to add it only to 
`&wdio-steps` to have a profiling build for linux node LTS like the wdio tests?

![image](https://user-images.githubusercontent.com/20856381/67114766-f65e8700-f1dc-11e9-8f6f-41e058393c0a.png)

Reply:
Good idea @pathmapper I've updated the PR